### DISCUSSION TITLE
For solo: open deck mode, changing the game to a perfect-information puzzle

### DISCUSSION THREAD

**Discussion by kevinb9n** [bafolts/terraforming-mars#2106] [Ideas]
I recently discovered the strange pleasure of replaying a solo game multiple times until I finally won it – nope – won it – nope – well lost it by less that time.

Strange because I knew every card I was going to see. Is this a common thing some players like to do?

If so, why not go whole-hog and just make the contents of the deck inspectable at any time? You don't feel tempted to jot down the list for yourself, or to use Silly Undo Tricks with Hi-Tech Lab and whatnot, and you don't even have to go through one throwaway blind game first per setup, just get to it.

I'm curious how many solo players would want something this, or only want to play hidden-deck solo (i.e. wouldn't even be sad if undo wouldn't work for card draws anymore), or actually want an in-between thing that requires clever tricks for peeking the deck.

**Comment by SliceOfBread** [comment]
I would say my favorite solo games are the ones I lose so badly that I wonder if they are impossible. I replay those until I win. I had a thread on BGG (https://boardgamegeek.com/thread/2427824/solo-challenge) with this idea. It did not get much traction but in an app version it might get more.

I will say, from doing this several times, I noticed that no matter how many times I did 'draw cards' actions, I could not affect when I got cards in research. I found that while research cards are dealt from the top of the deck, draws are dealt from the bottom. I think this makes sense for multiplayer games (all players will always get the same research cards every generation) but for solo it doesn't. I changed it in my own copy of the game.

**Comment by kevinb9n** [comment]
Good point, the deck being a deque (ha) doesn't make sense for solo. You're trying to use Business Contacts and such to get through the chaff to the cards you want, but why have Research cards be immune from that.

Is there a particular reason they wouldn't make this change here? Old seeds are thrown out daily, right?

**Comment by kevinb9n** [comment]
As emerged under #840, it's nice if most solo games were to go one way or the other: either open-deck or closed-deck (i.e. address #1846, but in the meantime, at least disable undo). For a given seed, the high score from each has meaning.

**Comment by kevinb9n** [comment]
Asking #solo-challenge what they thought about this yielded only one response, an extended pretty-negative one. No one happened to speak up and say they'd want to play it.
